assessment.rb

```diff
@@ -1,12 +1,32 @@
 require 'csv'
 # Assessment answers to:
+rails g model Licensee surname:string given_name:string gender:string height:integer weight:integer dob:date,eye_color:string hair_color:string
 
 # question 2:
+caren = Licensee.create surname: 'Bishop', given_name: 'Caren', gender: 'f', height:63, wieght:132, dob:1943-09-26, eye_color:'Brown', hair_color:'Black'
 
 # question 3:
+rails g model CreateLicensees
+require 'data/licensees.csv'
+class CreateCreateLicensees < ActiveRecord::Migration
+  def change
+    create_table :create_licensees do |t|
 
-# question 4:
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
I think you may have misunderstood this question. This is a migration; migrations change the structure of a database, not its contents. What we want here is a script for generating new instances of Licensee using the data found in data/licensees.csv